presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoResultSet.java

```diff
@@ -1863,7 +1863,7 @@ private static List<ColumnInfo> getColumnInfo(List<Column> columns)
                     .setTableName("") // TODO
                     .setColumnLabel(column.getName())
                     .setColumnName(column.getName()) // TODO
-                    .setColumnTypeSignature(parseTypeSignature(column.getType().toUpperCase(ENGLISH)))
+                    .setColumnTypeSignature(parseTypeSignature(column.getType().toLowerCase(ENGLISH)))
```

Comment:
What are the consequences if we do not do any case conversion for types?

Reply:
If we leave getName() without converting its result to lowercase (ENGLISH locale) as we have done in our method getFieldMap, it can potentially lead to case sensitivity issues, especially when dealing with column names in a case-insensitive context. 

If getName() returns column names with varying cases (e.g., "Name", "name", "NAME"), treating them as different keys in a map can lead to unintended behavior. 

By converting the column names to lowercase (ENGLISH locale), we ensure that the keys in your Map are consistent in terms of case.